content/en/docs/configuration/device-provisioning-service.md

```diff
@@ -14,12 +14,12 @@ The Device Provisioning Service provides API to provision device to the [plgd/hu
 Before you use the image you need to setup [K8s access to private registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry).
 
 ```bash
-docker pull ghcr.io/plgd-dev/device-provisioning-service/device-provisioning-service:latest
+docker pull ghcr.io/plgd-dev/hub/device-provisioning-service:latest
```

Comment:
This doesn't work right know, since we haven't made a realease yet with DPS.

Reply:
We can release it now :)